src/components/Banner.jsx

```diff
@@ -48,25 +53,32 @@ export const BannerWrapper = styled.div`
     display: flex !important;
     justify-content: center;
     align-items: center;
-    height: 125px !important;
+    height: 125px;
   }
   .card {
     flex-grow: 1;
-    height: 100% !important;
-  }
-  .card-content {
-    height: 100% !important;
+    height: 100%;
   }
   .slide {
-    height: 100% !important;
+    height: inherit;
```

Comment:
What height value is it supposed to inherit? If the purpose is to fill up the card, it should be something like 100%. Can you explain?

Reply:
It should inherit the value of .card of which it is a child. Both work equally well